src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/styles/style.css">
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <img
+        src="/images/imac.jpeg"
+        alt="iMac desktop"
+        class="card__image"
+      >
+      <h1 class="card__title">APPLE A1419 iMac 27&#34; Retina 5K Monoblock &#40;MNED2UA&#47;A&#41;</h1>
+      <p class="card__code" >Product code&#58; 195434</p>
+      <div class="card__rate">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+
```

Comment:
I suppose dat you can delete the empty lines under 'stars stars--4' block and add the empty lines after all main elements in your main block 'card'

Reply:
I did it according to checklist, statement 3. :)